DataFormats/Provenance/interface/ProductDescription.h

```diff
@@ -0,0 +1,9 @@
+#if !defined(DataFormats_Provenance_ProductDescription_h)
```

Comment:
I would have preferred
```suggestion
#ifndef DataFormats_Provenance_ProductDescription_h
```
but let's leave it later (if ever)